docker-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+NAME="mtasa"
+
+docker build . -t $NAME
+ID = $(docker images -q $NAME)
```

Comment:
Why do you need that and how to use it?

Reply:
When you execute the script, it will build the docker image and print its id

Reply:
We don't need that ID usually

Reply:
Its needed when you want to fully terminate a docker session

Reply:
Container tags are used for that.
```sh
docker stop <tag>
docker rm <tag>
```